doc/README.md

```diff
@@ -91,3 +90,7 @@ Here are some suggestions for http client packages:
 
 - [http](https://pub.dev/packages/http): A simple pacakage for perfoming http requests.
 - [Dio](https://pub.dev/packages/dio): A popular and powerful package for perfoming http requests.
+
+[awesome flame
+repository](https://github.com/flame-engine/awesome-flame#user-content-articles--tutorials)
```

Comment:
Weird formatting?

Reply:
Line length so it gets truncated.  This is just for the long links as per @st-pasha request to put long links at the bottom of the docs.